pkg/topology/kademlia/kademlia.go

```diff
@@ -40,7 +40,7 @@ const (
 var (
 	saturationPeers             = 8
 	overSaturationPeers         = 16
-	bootnodeOverSaturationPeers = 64
+	bootnodeOverSaturationPeers = 32
```

Comment:
Let's go more radical and go for 20.

Only thing I want to be sure of is what happens if we hit that limit. Will be block all new connections, or will we allow new connections and then start a process of deleting the old ones?

Reply:
We pick a random already connected peer, kick it, then connect to the new peer.